html/statbrowser.js

```diff
@@ -42,20 +42,152 @@ var turfcontents = [];
 var turfname = "";
 var imageRetryDelay = 500;
 var imageRetryLimit = 50;
+var topButtons = document.getElementById("top-buttons");
 var menu = document.getElementById("menu");
+var endMenu = document.getElementById("end-menu")
 var under_menu = document.getElementById("under_menu");
 var statcontentdiv = document.getElementById("statcontent");
 var storedimages = [];
 var split_admin_tabs = false;
 
+let toLookUp = [
+	"icon-size",
+	"text-mode",
+	"zoom-mode"
+]
+var lookedUpProperties = {}
+
+let bigButtons = [
+	{
+		name: "Changelog",
+		command: "Changelog",
+		class: "changelog",
+		icon: "list-ul"
+	},
+	{
+		name: "Rules",
+		command: "rules",
+		icon: "scale-balanced"
+	},
+	{
+		name: "Wiki",
+		command: "wiki",
+		icon: "book"
+	},
+	{
+		name: "Forum",
+		command: "forum",
+		icon: "envelope"
+	},
+	{
+		name: "Submit Bug",
+		command: "submit-bug",
+		class: "bug-button",
+		icon: "bug"
+	},
+	{
+		name: "Discord",
+		command: "discord",
+		class: "discord-button",
+		icon: "comments"
+	}
+]
+
+let clientButtons = {
+	"File": [
+		{name: "Take Screenshot	(F2)", command: ".screenshot auto"},
+		{name: "Save Screenshot As (Shift+F2)", command: ".screenshot"},
+		{divider: true},
+		{name: "Reconnect", command: ".reconnect"},
+		{name: "Options & Messages", command: ".options"},
+		{name: "Quit", command: ".quit"},
+	],
+	"Icons": [
+		{
+			name: "Stretch to fit",
+			command: ".winset \"mapwindow.map.icon-size=0\"",
+			active: "icon-size",
+			value: "0"
+		},
+		{
+			name: "256x256 (4x)",
+			command: ".winset \"mapwindow.map.icon-size=256\"",
+			active: "icon-size",
+			value: "256"
+		},
+		{
+			name: "128x128 (3x)",
+			command: ".winset \"mapwindow.map.icon-size=128\"",
+			active: "icon-size",
+			value: "128"
+		},
+		{
+			name: "96x96 (2.5x)",
+			command: ".winset \"mapwindow.map.icon-size=96\"",
+			active: "icon-size",
+			value: "96"
+		},
+		{
+			name: "64x64 (2x)",
+			command: ".winset \"mapwindow.map.icon-size=64\"",
+			active: "icon-size",
+			value: "64"
+		},
+		{
+			name: "32x32",
+			command: ".winset \"mapwindow.map.icon-size=32\"",
+			active: "icon-size",
+			value: "32"
+		},
+		{ divider: true },
+		{
+			name: "Text Mode",
```

Comment:
```suggestion
			name: "Text Mode",
```
This too, why do we even have this?